charts/seed-bootstrap/templates/curator-es/curator-configmap.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: curator-config
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: curator
+    role: logging
+data:
+  action_file.yml: |-
+    # Remember, leave a key empty if there is no value.  None will be a string,
+    # not a Python "NoneType"
+    #
+    # Also remember that all examples have 'disable_action' set to True.  If you
+    # want to use this action as a template, be sure to set this to False after
+    # copying it.
+    actions:
+      1:
+        action: delete_indices
+        description: "Clean up ES by deleting old indices"
+        options:
+          timeout_override:
+          continue_if_exception: False
+          disable_action: False
+          ignore_empty_list: True
+        filters:
+        - filtertype: age
+          source: name
+          direction: older
+          timestring: '%Y.%m.%d'
+          unit: months
+          unit_count: 1
+          field:
+          stats_result:
+          epoch:
+          exclude: False
+  config.yml: |-
+    # Remember, leave a key empty if there is no value.  None will be a string,
+    # not a Python "NoneType"
+    client:
+      hosts:
+        - elasticsearch-logging
```

Comment:
Indent it so it is like:

```yaml

      hosts:
      - elasticsearch-logging
```

Reply:
According to:
https://www.elastic.co/guide/en/elasticsearch/client/curator/current/configfile.html
the current state is correct.

Reply:
Both are valid.